src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -1413,4 +1413,70 @@ BOOST_AUTO_TEST_CASE(BRNNTrainReturnObjective)
   BOOST_REQUIRE_EQUAL(std::isfinite(objVal), true);
 }
 
+/**
+ * Test that RNN::Train() does not give an error for large rho.
+ */
+BOOST_AUTO_TEST_CASE(LargeRhoValueRnnTest)
+{
+  const int rho = 18;
```

Comment:
It's probably worth pointing out in a comment that it is intentional that this `rho` is larger than the maximum sequence length.  In fact, you could change it to 100 if you wanted (anything larger than the maximum sequence length should work).